Sources/Spezi/Dependencies/DependencyPropertyWrapper.swift

```diff
@@ -97,11 +97,18 @@ extension _DependencyPropertyWrapper: ModuleArrayDependency where Value == [any
     public convenience init() {
         self.init(DependencyCollection())
     }
-
+    
+    /// Creates the `@Dependency` property wrapper from an instantiated ``DependencyCollection``.
+    /// - Parameters:
+    ///    - dependencies: The ``DependencyCollection`` to be wrapped.
+    public convenience init(using dependencies: DependencyCollection) {
```

Comment:
Nice, I get why the addition is made, but I would suggest that we add a small motivating example to both newly implemented public functions either with an example or a short note where they can be used to other developers understand how they can be used, e.g., to implements like in Spezi LLM 👍

Reply:
Thanks for the suggestion, added a complete, coherent implementation example 👍